vulcanai/models/layers.py

```diff
@@ -120,9 +122,7 @@ def __init__(self, in_features, out_features,
         # Main layer
         self._kernel = nn.Linear(
                             in_features=self.in_features,
-                            out_features=self.out_features,
-                            bias=True
```

Comment:
you're removing the `bias=True` because that's the default value right? so it's redundant? just want to make sure

Reply:
yes.